src/main/java/org/example/PaperBook.java

```diff
@@ -0,0 +1,46 @@
+package org.example;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString(callSuper = true)
+public class PaperBook extends book {
+    private int pageCount;
+    private int widthInMm;
+    private int heightInMm;
+    private String publisher;
+    private String author;
+
+    public PaperBook(String title, String author, String publisher, int year, String genre, int pageCount, int widthInMm, int heightInMm) {
+        super(title, author, publisher, year, genre);
+        setTitle(title);
+        setAuthor(author);
+        setPublisher(publisher);
+        setYear(year);
+        setGenre(genre);

```

Comment:
this calls are redundant - you 've already passed (title, author, publisher, year, genre) to parent constructor